internal/cli/cli.go

```diff
@@ -111,16 +113,38 @@ func (c *banzaiContext) SetOrganizationID(id int32) {
 func (c *banzaiContext) save() {
 	log.Debug("writing config")
 
-	if err := viper.WriteConfig(); err != nil {
-		log.Infof("failed to write config: %v", err)
+	if viper.ConfigFileUsed() == "" {
+		log.Debug("no config file defined, falling back to default location $HOME/.banzai")
+
 		home, _ := homedir.Dir()
 		configPath := path.Join(home, ".banzai")
-		os.MkdirAll(configPath, os.ModePerm)
+		err := os.MkdirAll(configPath, os.ModePerm)
+		if err != nil {
+			log.Fatal(emperror.Wrap(err, "failed to create config dir"))
+		}
+
 		configPath = path.Join(configPath, "config.yaml")
```

Comment:
```suggestion
		configPath = filepath.Join(configPath, "config.yaml")
```
Filepath is compatible with the target operating system.